### ISSUE TITLE
App crash at /pods

### ISSUE THREAD

**Issue by ksa-real** [kontena/lens#5490]
**Describe the bug**
I clicked on one of my pods and had a crash

```
Error stack:

RangeError: Invalid string length
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467171:55)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
    at stringifySimple (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:467169:23)
```

```
Component stack:


    at MonacoEditor (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:587408:9)
    at div
    at div
    at DrawerParamToggler (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:580309:9)
    at span
    at div
    at DrawerItem (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:579924:23)
    at PodDetailsAffinities (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:534878:1)
    at div
    at PodDetails (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:536781:9)
    at div
    at div
    at div
    at Animate (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:573836:9)
    at NonInjectedDrawer (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:580067:9)
    at KubeObjectDetails (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:584658:9)
    at ErrorBoundary (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:582238:9)
    at NonInjectedClusterFrame (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/chunks/888.js:2924:9)
    at ThemeProvider (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:127160:24)
    at Router (http://fad75c7c21d3211bc4927b2eee98e27f.localhost:56951/build/Lens.js?199da6d1e6f9f3aa156b:454780:30)
```

**Comment by brickyard** [comment]
Also the same error.  Noticed after most recent update.  Unfortunately this renders Lens pretty unusable at the moment.

**Comment by ksa-real** [comment]
Also, for many pods the edit works but with a very large delay, like 10-15 seconds. Had to temporarily switch to `k9s` and pure `kubectl`.

**Comment by jihwan-m** [comment]
Also same error too. reconnecting to the cluster or restarting Lens does not help.

I'm using macOS Monterey, Apple M1 Pro.

**Comment by vothanhkiet** [comment]
I also have the same experience. Access the Edit pod button it will crash.

**Comment by jihwan-m** [comment]
I temporarily downgrade to v5.3.4 and it works.

**Comment by akshatjaindev** [comment]
@kumass2020 please share the v5.3.4 download link to downgrade.
I thought it is happening only to me.

**Comment by chenhunghan** [comment]
Should fix it https://github.com/lensapp/lens/pull/5479

**Comment by Nokel81** [comment]
Yes, fixed by #5479, new patch release will be released today including that fix.
